pkg/crc/machine/machine.go

```diff
@@ -573,7 +573,7 @@ func createHost(api libmachine.API, driverPath string, machineConfig config.Mach
 	}
 
 	if err := api.Create(vm); err != nil {
-		return nil, fmt.Errorf("Error creating the VM. %s", err)
+		return nil, fmt.Errorf("Error creating VM: %s", err)
```

Comment:
why remove `the` ?

Reply:
I was hoping there'd be > 1 VMs. the is back.

Reply:
The tool itself does not prevent the possibility, and this can be overriden, but the resource usage prevents this from being optimal.